### PR TITLE
added snmp checks

### PR DESCRIPTION
**Pull request by nixwiz**
Signed-off-by: Todd Campbell <todd@sensu.io>

### CODE REVIEW

File: snmp/metrics-snmp-bulk.rb [hunk 1]
```diff
@@ -0,0 +1,139 @@
+---
```
Comment (calebhailey) [inline snmp/metrics-snmp-bulk.rb]: s/.rb/.yaml/

**Review by calebhailey — COMMENTED** [review]
Just one file extension to fix here, then this one is ready to go. 👍 

Thanks!

**Comment by calebhailey** [comment]
One other thought here RE: patterns... we have an opportunity to diverge from the [upstream/downstream?] community plugins in this project. The configuration templates in this project shouldn't be a 1:1 mapping to plugins in https://github.com/sensu-plugins – they should be grouped by function/service. In this case, this PR adds templates for SNMP monitoring... could we also include some examples that use the excellent Nagios `check_snmp` plugin (which is available in https://bonsai.sensu.io/assets/sensu/monitoring-plugins)? 

This could be a separate PR, but I'd be interested to use SNMP as an example of how we plan to deviate from a single source of plugins.

**Comment by nixwiz** [comment]
> One other thought here RE: patterns... we have an opportunity to diverge from the [upstream/downstream?] community plugins in this project. The configuration templates in this project shouldn't be a 1:1 mapping to plugins in https://github.com/sensu-plugins – they should be grouped by function/service. In this case, this PR adds templates for SNMP monitoring... could we also include some examples that use the excellent Nagios `check_snmp` plugin (which is available in https://bonsai.sensu.io/assets/sensu/monitoring-plugins)?
> 
> This could be a separate PR, but I'd be interested to use SNMP as an example of how we plan to deviate from a single source of plugins.

Agreed, and we may also need to find a way to differentiate them somewhat.  For example, this has a check named 'check-snmp' and if we add the monitoring-plugins version, what name do we give it,  'monitoring-plugins-check-snmp' with subscriptions for 'snmp' and 'monitoring-plugins'?

Maybe we should consider the source for all when creating resource names?  Possibly needs a larger discussion.

**Pull request merged**